src/libraries/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoCurrencyGroupSizes.cs

```diff
@@ -10,21 +10,44 @@ public class NumberFormatInfoCurrencyGroupSizes
     {
         public static IEnumerable<object[]> CurrencyGroupSizes_TestData()
         {
-            yield return new object[] { NumberFormatInfo.InvariantInfo, new int[] { 3 } };
-            yield return new object[] { CultureInfo.GetCultureInfo("en-US").NumberFormat, new int[] { 3 } };
+            yield return new object[] { NumberFormatInfo.InvariantInfo, new int[] { 3 }, null };
+            yield return new object[] { CultureInfo.GetCultureInfo("en-US").NumberFormat, new int[] { 3 }, null };
 
             if (PlatformDetection.IsNotUsingLimitedCultures && !PlatformDetection.IsUbuntu && !PlatformDetection.IsWindows7 && !PlatformDetection.IsWindows8x && !PlatformDetection.IsFedora)
             {
-                yield return new object[] { CultureInfo.GetCultureInfo("ur-IN").NumberFormat, new int[] { 3, 2 } };
+                yield return new object[] { CultureInfo.GetCultureInfo("ur-IN").NumberFormat, new int[] { 3, 2 },  new int[] { 3 }};
             }
         }
 
+        private static bool ArrayEqual<T>(T[] a, T[] b)
+        {
+            if (a == null || b == null)
+            {
+                return a == b;
+            }
+
+            if (a.Length != b.Length)
```

Comment:
I think you can replace this whole section with `a.SequenceEquals(b)`

Reply:
(or `a.AsSpan().SequenceEqual(b.AsSpan())` if you want to avoid LINQ)